Fixes for open close connection for hw access functions

embedded-solutions opened the pull request.

skibool committed:
Fixes for open close connection for hw access functions

witczenko commented on microdaq/macros/microdaq_macros/mdaq_ao_scan_init.sci:
```diff
@@ -64,6 +64,7 @@ function  mdaq_ao_scan_init(arg1, arg2, arg3, arg4, arg5, arg6, arg7)
     end
 
     if scan_time < 0 then
```
To make sense for this warning line below,  the condition should be like this: 
if scan_time < 0 & scan_time <> -1 then

embedded-solutions replied:
This warning is only to notify the user that he should use -1 instead of value < then 0 - is rather to keep good function call values.

embedded-solutions replied:
ok, condition will be added

witczenko reviewed (commented):
Almost approve :), change warning or if condition, i leave you a comment

skibool committed:
Use same error when unable to connect to MicroDAQ

skibool committed:
Fix condition for worning when duration < 0 but not = -1

skibool committed:
Don't print error description from mlink_connect - upper layer should handle this

skibool committed:
Change info when model has finished execution to more acurate

skibool committed:
Remove double error printing

Use mdaq_error() instead of disp(mdaq_error())

skibool committed:
Fix mdaq_dsp_start

Now function uses error instead of printing error description. In case
of error script containging mdaq_dsp_start will be interrupted

skibool committed:
Fix error desciption in mdaq_open

embedded-solutions committed:
Merge branch '1.2v' into open_close_handling
